app/static/settings/translations.json

```
@@ -263,4 +263,37 @@
         "dark": "අඳුරු",
         "system": "පද්ධතිය"
     }
+    "lang_cs": {
+        "search": "Hledat",
+        "config": "Konfigurace",
+        "config-country": "Filtrovat výsledky podle země",
+        "config-country-help": "Poznámka: Pokud je povoleno, webová stránka se objeví ve výsledcích vyhledávání, pouze pokud je *hostována* ve vybrané zemi.",
+        "config-lang": "Jazyk rozhraní",
+        "config-lang-search": "Jazyk vyhledávání",
+        "config-near": "Poblíž",
+        "config-near-help": "Název města",
+        "config-block": "Blokovat",
+        "config-block-help": "Čárkami oddělený seznam stránek",
+        "config-theme": "Motiv",
+        "config-nojs": "Zobrazit NoJS odkazy",
+        "config-dark": "Tmavý motiv",
+        "config-safe": "Bezpečné vyhledávání",
+        "config-alts": "Nahradit odkazy na sociální média",
+        "config-alts-help": "Nahradí odkazy na Twitter, YouTube, Instagram atd. alternativami respektujícími soukromí.",
+        "config-new-tab": "Otevírat odkazy na novém listu",
+        "config-images": "Vyhledávání obrázků v plné velikosti",
+        "config-images-help": "(Experimentální) Přidá volbu ‚Zobrazit obrázek‘ do vyhledávání obrázků na ploše. Způsobí to, že náhledy výsledků vyhledávání obrázků budou mít nižší rozlišení.",
+        "config-tor": "Používat Tor",
+        "config-get-only": "Pouze požadavky GET",
+        "config-url": "Kořenová adresa URL",
+        "config-css": "Vlastní CSS",
+        "load": "Načíst",
+        "apply": "Použít",
+        "save-as": "Uložit jako...",
+        "github-link": "Zobrazit na GitHub",
+        "translate": "Přeložit",
+        "light": "Světlý",
+        "dark": "Tmavý",
+        "system": "Systémový"
+    },
```

Remove comma after this entry, since it's the last one

```suggestion
    }
```